### PR TITLE
Update package versions and tox environment list for Python 2

### PR DESCRIPTION
**Pull request by smcd253**
* change version limits for pyyaml and docker-compose to be compatible with Python 2 for Task
* add in py27 and py36 for tox env list

### CODE REVIEW

**Review by marianan — APPROVED** [review]
:shipit:

File: iotedgehubdev/__init__.py [hunk 1]
```diff
@@ -8,6 +8,6 @@
 pkg_resources.declare_namespace(__name__)
 
 __author__ = 'Microsoft Corporation'
-__version__ = '0.14.7'
+__version__ = '0.14.8rc0'
```
Comment (mhshami01) [inline iotedgehubdev/__init__.py:11]: consider using 0.14.7rc1?

**Review by cindydeng1998 — APPROVED** [review]
🛹

File: iotedgehubdev/__init__.py (hunk 1 above)
Comment (smcd253) [reply iotedgehubdev/__init__.py:11]: @mhshami01 this is out of scope for 0.14.7

**Review by mhshami01 — APPROVED** [review]

**Review by mhshami01 — APPROVED** [review]

**Pull request closed without merging**